src/autotype/AutoType.cpp

```diff
@@ -415,7 +415,7 @@ QList<AutoTypeAction*> AutoType::createActionFromTemplate(const QString& tmpl, c
     else if (tmplName == "ptrsc") {
         list.append(new AutoTypeKey(Qt::Key_Print));
     }
-    else if (tmplName == "scolllock") {
+    else if (tmplName == "scrolllock") {
```

Comment:
This appears to be a public API. If someone wanted to support both the misspelling and the correct spelling, that's definitely possible.